lisa/tools/gpu_smi.py

```diff
@@ -28,7 +28,6 @@ class NvidiaSmi(GpuSmi):
         ("H100", "44453233", 0),
         ("A10-4Q", "56475055", 0),
         ("A10-8Q", "3e810200", 0),
-        ("GB200", "42333130", 0),
```

Comment:
The ID mentioned here is a Mellonox one but not a GB200's. It should be removed as in this PR.